app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt

```diff
@@ -67,6 +70,8 @@ class NewConversationViewModel
 
     private var innerSearchPeopleState: SearchPeopleState by mutableStateOf(SearchPeopleState())
 
+    var self by mutableStateOf<SelfUser?>(null)
```

Comment:
we are using "kind" of :P MVI pattern a MVI pattern emits an updated status of a state object in thsi ViewModel it would be SearchPeopleState, so self should be merged inside the SerachPeopleState

Reply:
👍 sorry for that I'm not yet super-familiar with patterns and I saw something like I did in other places 😉 of course I will change that